### DISCUSSION TITLE
background-color, margin, etc on text

### DISCUSSION THREAD

**Discussion by aaaaargZombies** [uxbox/uxbox#1355] [Ideas]
This was raised as a Q+A here https://github.com/penpot/penpot/discussions/1160#discussion-3541425

It's been raised as a feature request before but I wanted to add a +1.

Figma call it auto-layout but essentially some mechanism for designing that relates to the way the design will be implemented on a webpage is incredibly useful, both in terms of productivity when drawing but also in having designers/developers on the same page.

**Comment by jcklpe** [comment]
Update on this subject here: https://community.penpot.app/t/what-are-penpot-plans-for-autolayout/112
